assets/css/settings-styles.css

```diff
@@ -1152,6 +1152,81 @@ div.upgrade-table-description a,
 	color: #fcfbf7;
 }
 
+.card-container {
+    border-radius: 5px;
+    width: calc(100% - 30px);
+    display: flex;
+    flex-wrap: wrap;
+    gap: 30px;
+    padding: 15px;
+    background-color: #f0f0f0;
+	justify-content: center;
+	padding-bottom: 50px;
+}
+
+.card {
+	border-radius: 5px;
+	margin-top: 0;
+	width: 25%;
```

Comment:
The cards need a max width to prevent them from becoming huge on larger screens. Perhaps a wrapper with a max width might be useful. And use either `flex` or `grid` to make three columns within this wrapper. The max width of the wrapper should more or less correspond with the width of the features table above it. So i would suggest a max width of something between 900px and 1200px.

It would be good to let the cards all have the same height no matter the content. I believe `grid` would let you more easily achieve this. But I'll leave it up to you how to achieve this, as I have no preference. 

Also the grid should be responsive. So on smaller screens the cards should stack underneath each other.

Reply:
@Terminator-Barbapapa I already use grid system, and I set it up in a way that all elements will have the height of the most higher element, so that they all look the same. That also goes for the installed plugin styles as well. Grid is responsible so on smaller screens, they will be stacked.

Reply:
@BrunoPavlinic98 Currently cards with more content are longer than cards with less content:

<img width="1502" alt="Scherm­afbeelding 2024-08-23 om 13 11 39" src="https://github.com/user-attachments/assets/d4dc5655-63e7-46f7-accd-3a38182c470e">

Also the current grid stacks in a weird way because the content is centered. But it also allows for one card to jump to the next row. Because we are providing three options here I think it would work best if we keep the row of three in tact for as long as possible and then switch to a stacked layout where all cards are underneath each other on small screens.

<img width="846" alt="Scherm­afbeelding 2024-08-23 om 13 12 34" src="https://github.com/user-attachments/assets/df9b3a76-8f46-4a8e-832f-e6f6d33f0534">